app/pb_kits/playbook/props.rb

```diff
@@ -21,7 +21,7 @@ def prop(name)
     included do
       prop :id, default: nil
       prop :data, type: Playbook::Props::Hash, default: {}
-      prop :classname, default: ""
+      prop :classname, default: nil
```

Comment:
do we need to default to nil? Maybe this should be set in the props DSL

Reply:
Makes sense. I'll update the DSL, add a test assertion in the appropriate spot, and remove unneeded default declarations.